spec/services/openid_connect_attribute_scoper_spec.rb

```diff
@@ -96,6 +96,25 @@
       end
     end
 
+    context 'with profile:name scope' do
+      let(:scope) { 'openid profile:name' }
+
+      it 'includes name attributes' do
+        expect(filtered[:given_name]).to be_present
```

Comment:
this is probably more about me than anything, but I never know what to put for `given_name` -- first name?